executor/builder.go

```diff
@@ -3932,6 +3932,10 @@ func (builder *dataReaderBuilder) buildTableReaderBase(ctx context.Context, e *T
 	if err != nil {
 		return nil, err
 	}
+	isolationLevel := kv.SI
```

Comment:
'isolationLevel := kv.SI' change the isolationLevel to SI isolation no matter what its original value is. Of course, It's OK now, because the default isolation is SI and tidb-server only supports SI in this process. If tidb-server supports more isolation levels in this process, here will be a hidden bug.
There are many such modifications.